### PR TITLE
Print results in table format in pr-perf-test

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description

Printing Perf test results in tabular form - https://screenshot.googleplex.com/3wqXpyZ84x8wEoR.png

### Testing
- Manual - Run script on my VM (Working fine)
- UnitTest - N/A
- IntegrationTest - N/A

### CODE REVIEW

**Review by raj-prince — COMMENTED** [review]
1. Can we add a separator or empty row b/w rows of different file-sizes?
2. Run the python linter/beautify command on the python files.

File: perfmetrics/scripts/presubmit/fetch_results.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  # Fetching metrics from json file
+  fio_metrics_obj = FioMetrics()
+  data = fio_metrics_obj.get_metrics(argv[1])
+
+  # Fetching results in result.txt file
+  file = open("result.txt", "a")
+  # Iterating through data
+  for d in data :
+    # Print filesize only once
+    if d['params']['rw'] == "read":
```
Comment (raj-prince) [inline perfmetrics/scripts/presubmit/fetch_results.py:36]: Please make this 1024 as const.

File: perfmetrics/scripts/presubmit/print_results.py [hunk 2]
```diff
@@ -12,30 +12,35 @@
...
+)
 
-  # Fetching results in result.txt file
-  file = open("result.txt", "a")
-  # Iterating through data
-  for d in data :
-    # Print filesize only once
-    if d['params']['rw'] == "read":
-      file.write(str.format("\n"+"Filesize: "+ str(round(d["params"]["filesize_kb"]/1024.0,3)) + "MiB" + "\n"))
+for i in range(0, 15, 5):
```
Comment (raj-prince) [inline perfmetrics/scripts/presubmit/print_results.py:25]: Please create variables for these numbers:
E.g.
data_dimension = 5 
data_set_split_index = 15 # data at [0, data_set_split_index) belong to Master branch and [data_set_split_index, size) belongs to PR branch.

File: perfmetrics/scripts/presubmit/fetch_results.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  if len(argv) != 2:
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 show_results.py <fio output json filepath>')
+
+  # Fetching metrics from json file
+  fio_metrics_obj = FioMetrics()
+  data = fio_metrics_obj.get_metrics(argv[1])
+
+  mb = 1024.0
```
Comment (raj-prince) [inline perfmetrics/scripts/presubmit/fetch_results.py:31]: Use capital letter for const.

File: perfmetrics/scripts/presubmit/print_results.py (hunk 2 above)
Comment (Tulsishah) [reply perfmetrics/scripts/presubmit/print_results.py]: fixed

File: perfmetrics/scripts/presubmit/fetch_results.py (hunk 3 above)
Comment (Tulsishah) [reply perfmetrics/scripts/presubmit/fetch_results.py]: fixed

File: perfmetrics/scripts/presubmit/print_results.py [hunk 4]
```diff
@@ -12,30 +12,38 @@
...
+  # Fetch Results for FileSize, Read, Write, RandRead, and RandWrite for Master
+  for j in range(0, DATA_DIMENSION):
+    dataMaster.append(data[i + j])
+  table.add_row(dataMaster)
+
+  dataPR = []
+  dataPR.append("PR")
+  # Fetch Results for FileSize, Read, Write, RandRead, and RandWrite for PR
+  for j in range(0, DATA_DIMENSION):
+    dataPR.append(data[i + j + 15])
```
Comment (raj-prince) [inline perfmetrics/scripts/presubmit/print_results.py:40]: Replace this 15 too.

Comment (Tulsishah) [reply perfmetrics/scripts/presubmit/print_results.py]: Fixed

**Pull request merged**